pep-0690.rst

```diff
@@ -479,11 +522,26 @@ enabled.
 C API
 -----
 
-For authors of C extension modules, the proposed
-``importlib.set_lazy_imports()`` function will also be exposed in the stable C
-API as ``PyImport_SetLazyImports(PyObject *names_or_callback_or_null)``, and
-``importlib.is_lazy_import`` will be available as ``PyDict_IsLazyImport(PyObject
-*dict, PyObject *key)``.
+For authors of C extension modules, the proposed public C API is as follows:
+
+* ``importlib.set_lazy_imports(enabled: bool = True, excluding: typing.Container[str] | None = None)``
```

Comment:
Is the `excluding` argument actually keyword-only? If so we should reflect that here:
```suggestion
* ``importlib.set_lazy_imports(enabled: bool = True, *, excluding: typing.Container[str] | None = None)``
```